packages/react-learn-card/src/components/VCDisplayCard2/VCDisplayCard2.tsx

```diff
@@ -0,0 +1,234 @@
+    handleXClick?: () => void;
+    getFileMetadata?: (url: string) => MediaMetadata;
+    getVideoMetadata?: (url: string) => VideoMetadata;
+    onMediaAttachmentClick?: (url: string) => void;
+    bottomRightIcon?: CredentialIconType;
+};
+
+export const VCDisplayCard2: React.FC<VCDisplayCard2Props> = ({
+    credential,
+    verificationItems,
+    issueeOverride,
+    issuerOverride,
+    subjectImageComponent,
+    issuerImageComponent,
+    verificationInProgress = false,
+    // convertTagsToSkills,
+    handleXClick,
+    getFileMetadata,
+    getVideoMetadata,
+    onMediaAttachmentClick,
+    bottomRightIcon,
+}) => {
+    const {
+        title = '',
+        createdAt,
+        issuer: _issuer = '',
+        issuee: _issuee = '',
+        imageUrl,
+    } = getInfoFromCredential(credential, 'MMM dd, yyyy');
+    const issuee = issueeOverride || _issuee;
+    const issuer = issuerOverride || _issuer;
+
+    const [isFront, setIsFront] = useState(true);
+    const [headerHeight, setHeaderHeight] = useState(79); // 79 is the height if the header is one line
+    const [headerWidth, setHeaderWidth] = useState(0);
+
+    const headerRef = useRef<HTMLHeadingElement>(null);
+
+    useLayoutEffect(() => {
+        // Needs a small setTimeout otherwise it'll be wrong sometimes with multiline header.
+        //   Probably because of the interaction with FitText
+        setTimeout(() => {
+            setHeaderHeight(headerRef.current?.clientHeight ?? 44);
+            setHeaderWidth(headerRef.current?.clientWidth ?? 0);
+        }, 10);
+    });
+
+    let worstVerificationStatus = verificationItems.reduce(
+        (
+            currentWorst: typeof VerificationStatusEnum[keyof typeof VerificationStatusEnum],
+            verification
+        ) => {
+            switch (currentWorst) {
+                case VerificationStatusEnum.Success:
+                    return verification.status;
+                case VerificationStatusEnum.Error:
+                    return verification.status === VerificationStatusEnum.Failed
+                        ? verification.status
+                        : currentWorst;
+                case VerificationStatusEnum.Failed:
+                    return currentWorst;
+            }
+        },
+        VerificationStatusEnum.Success
+    );
+
+    const statusColor = getColorForVerificationStatus(worstVerificationStatus);
+
+    const backgroundStyle = {
+        backgroundColor: credential.display?.backgroundColor,
+        backgroundImage: credential.display?.backgroundImage
+            ? `url(${credential.display?.backgroundImage})`
+            : undefined,
+        backgroundSize: 'contain',
+        backgroundPosition: 'center',
+        backgroundAttachment: 'fixed',
+    };
+
+    return (
+        <section className="font-mouse flex flex-col items-center border-solid border-[5px] border-white h-[700px] rounded-[30px] overflow-visible z-10 max-w-[400px] relative bg-white">
```

Comment:
Could you also add a custom class to this section, i noticed you have a fixed height here set via tailwind (700px), however I will need to override this to get it to display on smaller phones